AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -127,17 +128,12 @@ - (void)setConstrainedSize:(CGSize)constrainedSize
     // truncater do it's job again for the new constrained size. This is necessary as after a truncation did happen
     // the context would use the truncated string and not the original string to truncate based on the new
     // constrained size
-    __block ASTextKitContext *ctx = _context;
-    __block ASTextKitTailTruncater *tru = _truncater;
-    __block ASTextKitFontSizeAdjuster *adj = _fontSizeAdjuster;
+    [[ASRunLoopQueue sharedDeallocationQueue] enqueue:_context];
```

Comment:
Nit: Might be worth reading the deallocation queue only once here? Could we make a variant of the `enqueue` method that takes a variadic list of arguments?